README.md

```diff
@@ -26,7 +26,7 @@ Available Commands:
 
 Flags:
   -c, --clean              Remove intermediate and output directories
-      --context string     Input context name e.g. project.buildType+targetType
+      --context string     Input context name e.g. project[.buildType][+targetType]
```

Comment:
From [this requirement](https://github.com/Open-CMSIS-Pack/cbuild/issues/25#issue-1468149063) I believe the `[project]` part of the context string should be optional as well. In other words the `[project]` may be empty and the selected contexts should be handled accordingly.

Reply:
I agree, but only once we support to orchestrate multiple project build.

Reply:
Isn't multiple project build support being introduced in this PR?
From its [description](https://github.com/Open-CMSIS-Pack/cbuild/pull/35#issue-1566294565):
_From all contexts the projects with the filtered contexts shall be converted and built sequentially._

Reply:
@brondani  Yes, your observation is right. It has been introduced in this PR. With this PR, project contexts are built one after the other. Currently, the order of execution is not controlled.
I believe we should make the change related to ` [project]`.  It should be doable now.

Reply:
Apologies.

Reply:
Addressed here
[Support optional project name in [project][.buildType][+targetType]](https://github.com/Open-CMSIS-Pack/cbuild/pull/35/commits/1c4ab2cea399cdbef7ba0a42487020c39874e5d7)